armada-ths/ais
# Bugfix no company address

Set company address fields to required=False so that the bug with some companies not being able to save will hopefully be fixed. Also did som ui updates on login/reset password screen (changed from Bebas font to Lato in input so you can see if you're using lower or uppercase) and added placeholders to company address fields (they were there previously but must have dissappeared at some point).

This works locally, for me!

Test:
1. Be logged out. Look at login screen: The fonts in input for email and password should be Lato (aka not the same font as the title above the input field).
2. Same as above but reset password view. 
3. Login with a user connected to a company where the address fields are not filled in correctly (you can edit these in the admin view), try to save the CR, even though the address is not filled in correctly. 
4. Try to fill in the company address etc from the CR, is it obvious which info should be in which input field?
